### PR TITLE
STAR-14 add qr code to share event section

### PR DESCRIPTION
**Pull request by oleynichenko**
Screenshot (47)

### CODE REVIEW

File: src/components/event/dashboard/share.js [hunk 1]
```diff
@@ -1,10 +1,74 @@
...
+  color: ${(props) => props.theme.palette.primary.main};
+  cursor: pointer;
+`;
+
+const downloadQRCode = () => {
+  const qrCodeURL = document.getElementById('eventQrCode').toDataURL();
+  const aEl = document.createElement('a');
+
+  aEl.href = qrCodeURL;
+  aEl.download = 'qr_code.png';
```
Comment (vitalii-retel) [inline src/components/event/dashboard/share.js:42]: I think this name should be more specific. I can imagine I find `qr_code.png` in my download folder, and I have no idea what that code for.

**Review by vitalii-retel — COMMENTED** [review]
Look ok, I think we can use it for now. Not sure if it should be generated on server, but can be changed anytime.

File: src/components/event/dashboard/share.js (hunk 1 above)
Comment (oleynichenko) [reply src/components/event/dashboard/share.js]: fixed

File: src/components/event/dashboard/share.js [hunk 2]
```diff
@@ -1,10 +1,76 @@
...
+  color: ${(props) => props.theme.palette.primary.main};
+  cursor: pointer;
+`;
+
+const downloadQRCode = (id) => {
+  const qrCodeURL = document.getElementById('eventQrCode').toDataURL();
+  const aEl = document.createElement('a');
+
+  aEl.href = qrCodeURL;
+  aEl.download = `starfleet_qr_code_${id}.png`;
```
Comment (vitalii-retel) [inline src/components/event/dashboard/share.js:42]: well, it is not much better. But let's change it later.

**Review by vitalii-retel — APPROVED** [review]

**Pull request merged**